sdk/documenttranslator/ai-document-translator-rest/review/ai-document-translator.api.md

```diff
@@ -0,0 +1,892 @@
+## API Report File for "@azure-rest/ai-document-translator"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { Client } from '@azure-rest/core-client';
+import { ClientOptions } from '@azure-rest/core-client';
+import { HttpResponse } from '@azure-rest/core-client';
+import { KeyCredential } from '@azure/core-auth';
+import { RawHttpHeaders } from '@azure/core-rest-pipeline';
+import { RequestParameters } from '@azure-rest/core-client';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public (undocumented)
+export interface BatchRequest {
```

Comment:
please add documentation if possible.

Reply:
😢  swagger doesn't have a description for this